### DISCUSSION TITLE
Current implementation of fit_predict() and OxCGRT-parameters forecast

### DISCUSSION THREAD

**Discussion by Inglezos** [lisphilar/covid19-sir#505] [General]
I noticed that with the current implementation of parameters forecast sometimes the predicted `Rt` value can become negative if delay is large. For example I tried hardcoded delay of 30 days instead of the recovery period and I run the estimator for Greece. The result was either to have a phase with 0 length in days or `Rt` to become small negative. Is it a bug or is this happening because the mechanism is not fully developed yet?

**Comment by lisphilar** [comment]
This is caused by unknown reasons and please create a bug-fix issue with the details.
Is rho value also negative?

**Reply by Inglezos** [reply]
Yes I think the `rho` values were also negative.

**Reply by Inglezos** [reply]
I created the bug issue #507.

**Reply by lisphilar** [reply]
Thank you. We do not have "close" feature "Discussions" beta version unfortunately.
